client/layout/guided-tours/tours/media-basics-tour.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+	Continue
+} from 'layout/guided-tours/config-elements';
+import {
+	doesSelectedSiteHaveMediaFiles,
+	isNewUser,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const MediaBasicsTour = makeTour(
+	<Tour
+		name="mediaBasicsTour"
+		version="20170321"
+		path="/media"
+		when={ and(
+			isDesktop,
+			isNewUser,
+			doesSelectedSiteHaveMediaFiles,
+		) }
+	>
+		<Step
+			name="init"
+			arrow="top-left"
+			target=".media-library__upload-buttons"
+			placement="below"
+		>
+			<p>
+				{ translate( 'Welcome to your media libary!' ) }
+			</p>
+			<p>
+				{
+					translate( 'Upload media — photos, documents, audio files, and more — ' +
+						'by clicking the {{icon/}} {{strong}}Add New{{/strong}} button.',
+						{
+							components: {
+								icon: <Gridicon icon="add-image" />,
+								strong: <strong />,
+							}
+						}
+					)
+				}
+			</p>
+			<ButtonRow>
+				<Next step="drag-and-drop" />
+				<Quit />
+			</ButtonRow>
+		</Step>
+
+		<Step
+			name="drag-and-drop"
+			placement="right"
+		>
+			<p>
+				{ translate( 'You can also drag-and-drop image and video files from your computer into your media library.' ) }
+			</p>
+			<img
+				src="https://en-support.files.wordpress.com/2017/07/media-drag-and-drop.gif"
```

Comment:
(thus: https://i0.wp.com/en-support.files.wordpress.com/2017/07/media-drag-and-drop.gif)